src/components/LiveChart/index.tsx

```diff
@@ -164,10 +164,10 @@ function LiveChart({
         console.log(error)
         setStateProChart({ hasProChart: false, pairAddress: '', apiVersion: '', loading: false })
       })
-    // eslint-disable-next-line react-hooks/exhaustive-deps
     return () => {
       currenciesChanged = true
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
LOL?

Reply:
revert code of Mr Diep.

Reply:
<img width="540" alt="image" src="https://user-images.githubusercontent.com/33005392/198187974-a168b4b2-1c06-48b2-a79c-9f37edd5f97b.png">